webapp/channels/src/sass/components/_buttons.scss

```diff
@@ -9,49 +30,221 @@ button {
         background: #f74343;
         border-radius: 50%;
     }
+}
 
-    &.style--none {
+.btn {
+    // Important is given to override bootstrap styles on different states
```

Comment:
This isn't necessary for this PR, but this seems like a good time for me to ask this. Bootstrap causes pain for us on the dev side, but I've never really asked the UX perspective about this. Does Bootstrap also cause issues for your team, and should we be focused more on removing it?

It feels to me like we only accidentally use Bootstrap classes, and we often end up having to work around it when we use the same, common class names. I know we used it intentionally in older code, so we'd need to migrate some stuff away from it, but it seems like it'd be good to do that. We're also on a very old version of Bootstrap if I remember right.

If UX is on also on board with removing it, we could discuss a way to do that over time since it might be possible to remove most of Bootstrap except for what we currently use, or we could pull out the useful styles and get rid of the library itself.

Reply:
Yes, it does cause a lot of issues. @hmhealey

Reply:
Cool, maybe we can find some time to work on that eventually. At the very least, I spent some time last week taking a look through which Bootstrap classes that we actually use either directly or via React Bootstrap, so I have a bit better idea of how much it'll be to clean those up later